src/core.py

```diff
@@ -59,6 +59,14 @@ def on_config(self, config):
                 TECHDOCS_DEFAULT_THEME,
             )
 
+        if "features" not in config["theme"]:
+            config["theme"]["features"] = []
+
+        config["theme"]["features"].append("navigation.footer")
+        config["theme"]["features"].append("content.action.edit")
+
+        config["theme"]["palette"] = ""
```

Comment:
What does this mean to set it to blank? anything from the changeset of new mkdocs-material version that we want to leave here for future context?

Reply:
Yeah it's not super obvious... 

Some background:
After patching a lot of style changes manually in `@backstage/plugin-techdocs`, I noticed that html files generated with `mkdocs-material 8.x` had the attribute `data-md-color-scheme=""` on the body tag, whereas the files generated with `mkdocs-material 9.x` had the attribute `data-md-color-scheme="default"`. 

When i set the palette to `""` in the config here, almost all of the patches I had done in `@backstage/plugin-techdocs` could be removed.

I haven't looked into all the details of the palette attribute, and since 

1. It keeps backwards compatibility in a simple way
2. If people want to alter colors, they should probably do so in backstage, and not in the mkdocs.yaml

I figured this was a reasonable solution.

But yes, I agree that we should add more for future context. I've added some more details to the changelog: 60f5c4ed6446d7e02e5c0be87228801f87c273be